planner/core/logical_plan_builder.go

```diff
@@ -576,6 +576,23 @@ func (b *PlanBuilder) buildProjectionField(id, position int, field *ast.SelectFi
 	}
 }
 
+func eliminateIfNullOnNonNullColumn(p LogicalPlan, expr expression.Expression) expression.Expression {
```

Comment:
We usually use `NotNull` instead of `NonNull`.

Reply:
s/NonNullColumn/NotNullCol/